rcl/CMakeLists.txt

```diff
@@ -3,11 +3,36 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rcl)
 
 find_package(ament_cmake REQUIRED)
+find_package(rcl_interfaces REQUIRED)
+find_package(rmw REQUIRED)
+find_package(rosidl_generator_cpp REQUIRED)
```

Comment:
I assume this will become rosidl_generator_cpp once Dirk finishes C typesupport?